### PR TITLE
Laba1

### PR DESCRIPTION
**Pull request by peopleAlreadyKnowWhoIAm**

### CODE REVIEW

File: itunes.sql [hunk 1]
```diff
@@ -0,0 +1,647 @@
...
+    REFERENCES `itunes`.`genre` (`id`)
+    ON DELETE RESTRICT
+    ON UPDATE CASCADE)
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `itunes`.`_playlist_has_song`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `itunes`.`_playlist_has_song` (
```
Comment (redd4ford) [inline itunes.sql]: is this even legal to start a table name with underscore in MySQL?

Comment (redd4ford) [reply itunes.sql]: Ok, I just checked and it is legal, but not a common practice.
However, you need to stick to your naming convention. In your database, no other table starts with an underscore, so please remove it here too.

File: itunes.sql [hunk 2]
```diff
@@ -0,0 +1,647 @@
...
+    ON UPDATE CASCADE)
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `itunes`.`user_credential`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `itunes`.`user_credential` (
+  `user_id` INT NOT NULL,
+  `password` VARCHAR(45) NOT NULL,
```
Comment (redd4ford) [inline itunes.sql:254]: Just so you know, typically you don't store the password in your DB, you store a salt (salt is an encrypted version of password that takes much longer to crack, and when a user logs in, you encrypt the password they provided and compare it with the salt stored) or use an external auth system (e.g. Firebase Auth, AWS Cognito etc.).

This is not required for this task, but just so you know for your future work experience.

File: itunes.sql [hunk 3]
```diff
@@ -0,0 +1,647 @@
...
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `itunes`.`user`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `itunes`.`user` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(45) NOT NULL,
+  `email` VARCHAR(45) NOT NULL,
```
Comment (redd4ford) [inline itunes.sql:122]: I believe email should be unique, no?

File: itunes.sql (hunk 1 above)
Comment (peopleAlreadyKnowWhoIAm) [reply itunes.sql]: Changed

File: itunes.sql (hunk 3 above)
Comment (peopleAlreadyKnowWhoIAm) [reply itunes.sql:122]: Changed

**Pull request merged**